units-databinding/src/main/kotlin/dev.icerock.moko.units.databinding/BindingAdapters.kt

```diff
@@ -95,3 +99,40 @@ private fun getLifecycleOwnerFromContext(context: Context): LifecycleOwner {
     if (context is ContextWrapper) return getLifecycleOwnerFromContext(context.baseContext)
     throw IllegalArgumentException("context $context not implement LifecycleOwner")
 }
+
+fun RecyclerView.bindUnits(lifecycleOwner: LifecycleOwner, units: LiveData<List<UnitItem>>) {
+    val unitsAdapter = UnitsRecyclerViewAdapter(lifecycleOwner)
+    adapter = unitsAdapter
+
+    units.observe(lifecycleOwner) {
+        if (units.value?.isNotEmpty() == true) {
+            unitsAdapter.units = it.orEmpty()
+        }
+    }
+}
+
+fun RecyclerView.bindPagingUnits(
+    lifecycleOwner: LifecycleOwner,
+    units: LiveData<List<TableUnitItem>?>,
+    nextPageAction: () -> Unit
+) {
```

Comment:
it's not databinding adapter. it's seems like it for viewbinding

Reply:
moved to units-viewbinding/.../RecyclerViewBinding.kt